lib/tasks/coverage.js

```diff
@@ -36,13 +36,20 @@ const configure = function configure (gulp, opts, env) {
           bins[item] = path.resolve(npmBin, item);
           return bins;
         }, {});
-        const args = ['--reporter=lcov', '--reporter=text-summary', `--report-dir=${targetDir}`,
-          bins._mocha, '--reporter=dot', ...files, '--exit'];
+        const args = [
+          '--reporter=text-lcov',
+          `--report-dir=${targetDir}`,
+          '--exclude-after-remap=false',
+          bins._mocha,
+          '--reporter=dot',
+          ...files,
+          '--exit',
+        ];
         let env = _.clone(process.env);
         env.NO_PRECOMPILE = 1;
         env._TESTING = 1;
         env.NODE_ENV = 'coverage';
-        log(`running command --> ${bins.nyc} ${args.join(' ')}`);
+        log(`Running command: ${bins.nyc} ${args.join(' ')}`);
```

Comment:
is nyc still used (I see it was removed from deps)?

Reply:
The config is removed from `package.json` and into the command line, otherwise it needs to be moved into the enclosing package to be run, which is a headache.